src/app/core/campaign/campaign.service.ts

```diff
@@ -13,8 +13,11 @@ export class CampaignService {
   constructor(private augury: AuguryService) {}
 
   getCampaign(id: number | string): Observable<CampaignState> {
-    return this.augury.follow('prx:campaign', { id, zoom: 'prx:flights' }).pipe(
-      switchMap(doc => doc.followItems('prx:flights').pipe(map(flightDocs => ({ doc, flightDocs })))),
```

Comment:
There's an error on staging right now due to a change in the API. This was loading the campaign with flights embedded with zoom. However, that embedded list of flights is unexpectedly an unpaged list and causes the Hal support to throw an error. So I changed it to load the flights separately. It now assumes that campaign/flights is a paged list by default, so it loads the paged list of flights to get the total amount, then loads the full list of flights using total as the per param.